### ISSUE TITLE
Get Badge API on Roadmap

### ISSUE THREAD

**Issue by timothyfcook** [CityOfLearning/public#9]
Heard that the plan is to use Badgekit again? Saw [here:] (https://github.com/CityOfLearning/public/wiki/Domain-Objects) that a Badge API will be available soon, but didn't see it on Roadmap.

**Comment by migheljackson** [comment]
@teverette Does this fall under the Partner API roadmap item?

**Comment by timothyfcook** [comment]
@migheljackson good point. Seems like it could be part of that. Obviously Badges were a big part of the need for a Partner API, from what I understood. Speaking of, do you guys have some specific use cases you were planning for once the Partner API is up? Is it intended to enable interactions with other Badge issuers, or more for other platforms, e.g. Eventbrite, etc.?

Would be helpful to hear any examples you were considering.

**Comment by teverette** [comment]
In general the goal for Partner API is to allow existing system is easily integrate with the COL Platform without requirement additional work such as double entry of students and badges. 

And yes the Partner API roadmap item is around issuing, reading, and creating badges. There is also the component where the student authorizes the organization to access  their account (ie issue badges to their account). 

Further down the line, Partner API will extend the ability to add programs to the COL platform and learners, and to roster those learners to those programs.

However, a BadgeKit replacement technically will include a UI and that will be handled in an Org Dashboard update. Nobody volunteered to handle it, so It may be broken out from a org dashboard roadmap item. Make sense?
